File: Challenge_1.js

const list = [3,5,8,6,4,77]
const K = 14

function challenge(list, num){
    list.map((item, id)=>{
        for(var i=0; i < list.length; i++){
            if(item + list[i] === num){
                console.log(`${item} at ${id} and ${list[i]} at ${i} are equal to num`)
            }
        }
    })
}

challenge(list,K)

File: Challenge_1.py


list = [34,2,65,34,20,15,7,5]
K = 25

def challenge(list, num):
        for x in list:
            for y in list:
                if x + y == num:
                    print(x, ' ', y, ' are equal to num')


challenge(list, K)

